src/alert-manager/deploy/alert-manager-configmap.yaml.template

```diff
@@ -73,6 +77,13 @@ data:
       - url: 'http://localhost:{{ cluster_cfg["alert-manager"]["alert-handler"]["port"] }}/alert-handler/send-email-to-admin/?template=cluster-usage'
         send_resolved: false
       {% endif %}
+    
+    - name: pai-job-status-change
+      webhook_configs:
+      {% if 'email-user' in cluster_cfg["alert-manager"]["actions-available"] %}
+      - url: 'http://localhost:{{ cluster_cfg["alert-manager"]["alert-handler"]["port"] }}/alert-handler/send-email-to-admin/?template=job-status-change'
```

Comment:
Does `send-email-to-admin` just send mails to administrator? I think mails should be sent to corresponding users.

Reply:
> Does `send-email-to-admin` just send mails to administrator? I think mails should be sent to corresponding users.

Yes, it should be `send-email-email-to-user`